x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -120,249 +117,245 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
     }
   };
 
+  const rightPageHeaderButtons = hasEditButton
+    ? [
+        <>
+          <EuiButtonEmpty
+            data-test-subj="openEditAlertFlyoutButton"
+            iconType="pencil"
+            onClick={() => setEditFlyoutVisibility(true)}
+            name="edit"
+            disabled={!alertType.enabledInLicense}
+          >
+            <FormattedMessage
+              id="xpack.triggersActionsUI.sections.alertDetails.editAlertButtonLabel"
+              defaultMessage="Edit"
+            />
+          </EuiButtonEmpty>
+          {editFlyoutVisible && (
+            <AlertEdit
+              initialAlert={alert}
+              onClose={() => {
+                setInitialAlert(alert);
+                setEditFlyoutVisibility(false);
+              }}
+              actionTypeRegistry={actionTypeRegistry}
+              alertTypeRegistry={alertTypeRegistry}
+              onSave={setAlert}
+            />
+          )}
+        </>,
+      ]
+    : [];
+
   return (
-    <EuiPage>
-      <EuiPageBody>
-        <EuiPageContent>
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle size="m">
-                <h1>
-                  <span data-test-subj="alertDetailsTitle">{alert.name}</span>
-                </h1>
-              </EuiTitle>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiFlexGroup responsive={false} gutterSize="xs">
-                {hasEditButton ? (
-                  <EuiFlexItem grow={false}>
-                    <>
-                      {' '}
-                      <EuiButtonEmpty
-                        data-test-subj="openEditAlertFlyoutButton"
-                        iconType="pencil"
-                        onClick={() => setEditFlyoutVisibility(true)}
-                        name="edit"
-                        disabled={!alertType.enabledInLicense}
-                      >
-                        <FormattedMessage
-                          id="xpack.triggersActionsUI.sections.alertDetails.editAlertButtonLabel"
-                          defaultMessage="Edit"
-                        />
-                      </EuiButtonEmpty>
-                      {editFlyoutVisible && (
-                        <AlertEdit
-                          initialAlert={alert}
-                          onClose={() => {
-                            setInitialAlert(alert);
-                            setEditFlyoutVisibility(false);
-                          }}
-                          actionTypeRegistry={actionTypeRegistry}
-                          alertTypeRegistry={alertTypeRegistry}
-                          onSave={setAlert}
-                        />
-                      )}
-                    </>
-                  </EuiFlexItem>
-                ) : null}
+    <EuiPageBody>
+      <EuiPageContent color="transparent">
```

Comment:
Sorry @YulNaumenko , I'm having some trouble with my Gmail filters and this one slipped through. I think this actually still has one wrapper too many causing extra padding all around the content. You should be able to get rid of `EuiPageBody`.

Reply:
@cchaos could you please open the proper bug?

Reply:
I've got a branch going to fix a few inconsistencies in implementation of the template. I'll fix this there.